aether-ui/aether/ui/assets/apps/pipeline/components/DeleteModal.jsx

```diff
@@ -41,7 +41,7 @@ class DeleteModal extends Component {
       <span>
         <FormattedMessage
           id='delete.modal.header'
-          defaultMessage='{header}'
+          defaultMessage='{header} {objectType} '
```

Comment:
If we run the extract "translatable" texts script here (the one that we didn't include yet) the extracted text would be `"{header} {objectType} "`...